clang/runtime/dpct-rt/include/dpl_extras/vector.h

```diff
@@ -258,61 +337,75 @@ class device_vector {
   void shrink_to_fit(void) {
     if (_size != capacity()) {
       size_type tmp_capacity = ::std::max(_size, _min_capacity());
```

Comment:
If current capacity equal to `_min_capaticy()`, the following code can be jumped.